src/test/ui/tuple/wrong_argument_ice.stderr

```diff
@@ -3,14 +3,16 @@ error[E0061]: this function takes 1 argument but 2 arguments were supplied
    |
 LL |         self.acc.push_back(self.current_provides, self.current_requires);
    |                  ^^^^^^^^^ ---------------------  --------------------- supplied 2 arguments
-   |                  |
-   |                  expected 1 argument
    |
 note: associated function defined here
   --> $SRC_DIR/alloc/src/collections/vec_deque/mod.rs:LL:COL
    |
 LL |     pub fn push_back(&mut self, value: T) {
    |            ^^^^^^^^^
+help: use parentheses to construct a tuple
+   |
+LL |         self.acc.push_back((self.current_provides, self.current_requires));
```

Comment:
Even though I get different error messages on my local machine this is what the CI test gets, weird 🤔

Reply:
Weird. Maybe something changed on master? If something like this happens in the future, try rebasing onto the latest master

Reply:
Will do thanks for the tip